<!DOCTYPE html>
<html><head><title>Message</title><link rel="stylesheet" href="../../styles.css"/><script src="../../scripts.js"></script></head><body onload="ro();">
<div class="rH">Base:</div><div class="rH">10 references to Message</div><div class="rA">Azure.Messaging.EventHubs (2)</div><div class="rG" id="Azure.Messaging.EventHubs"><div class="rF"><div class="rN">EventHubsException.cs (1)</div>
<a href="../EventHubsException.cs.html#204"><b>204</b>$&quot;{ typeof(EventHubsException).FullName }({ Reason }): { <i>Message</i> }{ Environment.NewLine }{ StackTrace }&quot;;</a>
</div>
<div class="rF"><div class="rN">SharedSource\Processor\PartitionLoadBalancer.cs (1)</div>
<a href="../SharedSource/Processor/PartitionLoadBalancer.cs.html#550"><b>550</b>exception.SetFailureOperation(exception.<i>Message</i>);</a>
</div>
</div>
<div class="rA">Azure.Messaging.EventHubs.Processor (1)</div><div class="rG" id="Azure.Messaging.EventHubs.Processor"><div class="rF"><div class="rN">SharedSource\Processor\PartitionLoadBalancer.cs (1)</div>
<a href="../../Azure.Messaging.EventHubs.Processor/SharedSource/Processor/PartitionLoadBalancer.cs.html#550"><b>550</b>exception.SetFailureOperation(exception.<i>Message</i>);</a>
</div>
</div>
<div class="rA">Azure.Messaging.EventHubs.Shared.Tests (1)</div><div class="rG" id="Azure.Messaging.EventHubs.Shared.Tests"><div class="rF"><div class="rN">SharedSource\Processor\PartitionLoadBalancer.cs (1)</div>
<a href="../../Azure.Messaging.EventHubs.Shared.Tests/SharedSource/Processor/PartitionLoadBalancer.cs.html#550"><b>550</b>exception.SetFailureOperation(exception.<i>Message</i>);</a>
</div>
</div>
<div class="rA">Azure.Messaging.EventHubs.Tests (6)</div><div class="rG" id="Azure.Messaging.EventHubs.Tests"><div class="rF"><div class="rN">Errors\EventHubsExceptionTests.cs (5)</div>
<a href="../../Azure.Messaging.EventHubs.Tests/Errors/EventHubsExceptionTests.cs.html#94"><b>94</b>///   Verifies functionality of the &lt;see cref=&quot;EventHubsException.<i>Message</i>&quot; /&gt;</a>
<a href="../../Azure.Messaging.EventHubs.Tests/Errors/EventHubsExceptionTests.cs.html#106"><b>106</b>Assert.That(instance.<i>Message</i>, Is.EqualTo(message));</a>
<a href="../../Azure.Messaging.EventHubs.Tests/Errors/EventHubsExceptionTests.cs.html#110"><b>110</b>///   Verifies functionality of the &lt;see cref=&quot;EventHubsException.<i>Message</i>&quot; /&gt;</a>
<a href="../../Azure.Messaging.EventHubs.Tests/Errors/EventHubsExceptionTests.cs.html#121"><b>121</b>Assert.That(instance.<i>Message</i>, Does.Contain(namespaceValue), &quot;The message should include the Event Hubs namespace&quot;);</a>
<a href="../../Azure.Messaging.EventHubs.Tests/Errors/EventHubsExceptionTests.cs.html#122"><b>122</b>Assert.That(instance.<i>Message</i>, Does.Contain(message), &quot;The message should include the exception message text&quot;);</a>
</div>
<div class="rF"><div class="rN">Primitives\EventProcessorTests.MainProcessingLoop.cs (1)</div>
<a href="../../Azure.Messaging.EventHubs.Tests/Primitives/EventProcessorTests.MainProcessingLoop.cs.html#383"><b>383</b>expectedException.<i>Message</i>),</a>
</div>
</div>
</body></html>